inc/class-shortcode-ui.php

```diff
@@ -191,23 +190,33 @@ private function compare_shortcodes_by_label( $a, $b ) {
 	public function handle_ajax_do_shortcode() {
 
 		// Don't sanitize shortcodes — can contain HTML kses doesn't allow (e.g. sourcecode shortcode)
-		$shortcode = ! empty( $_POST['shortcode'] ) ? stripslashes( $_POST['shortcode'] ) : null;
```

Comment:
It looks like WPCS don't support ternaries for input validation. Maybe that feature should be added.